go/converters.go

```diff
@@ -1,12 +1,20 @@
 package odict
 
 import (
+	"bytes"
+	"embed"
 	"encoding/json"
 	"encoding/xml"
+	"fmt"
 
+	"github.com/bokwoon95/sq"
+	"github.com/bokwoon95/sqddl/ddl"
 	flatbuffers "github.com/google/flatbuffers/go"
 )
 
+//go:embed sql.go
```

Comment:
Just genuinely curious seeing I haven't read up on the library – what does this do?

Reply:
Not going to pretend to be a pro on this, but according to ChatGPT,

"The `//go:embed` directive is used to embed files in a Go binary at build time. In this code, the `sql.go` file is embedded as an asset, so it can be used later in the SQL function.

In the `SQL` function, `ddl.GenerateCmd` is used to generate SQL create statements and constraints, which are appended to a string `sqlCmds`. This string is built using the embedded `sql.go` file as a source for the SQL template.

Finally, the `sqlCmds` string is returned as the output of the SQL function."

From `sqddl`'s documentation, it seems this description is accurate. I found a reference to it here: https://bokwoon.neocities.org/sqddl#toc-loading-embedded-files.